.nvmrc

```diff
@@ -1 +1 @@
-lts/gallium
+lts/hydrogen
```

Comment:
> What else should I to ensure the changes wouldn't break anything?

We have some flexibility here, as we know we are the direct consumers of this - the package is public, but not recommended or endorsed for non Skyscanner usage.

We can release a new version, test it internally locally and on staging, and fix forward with a new release if it has issues.

Alternatively we could release a beta, text using the beta, then publish a stable version if the testing goes well.

There isn't documentation in this repo about betas (we could add it) but we can borrow from 
https://github.com/Skyscanner/eslint-config-skyscanner/blob/main/docs/releasing.md#alphabeta-release

Reply:
In this case - we're not actually making any functional changes, and we're already running on Node 18 internally, this PR just makes it the minimum (which is a breaking change, so a new major version). That means there is no need for a beta - we know the package functionally works.

Reply:
Thanks for the info! I'll publish a new major version then test the release.

Reply:
I think we could also test via commit hash without doing a release?